### PR TITLE
Custom Editors for FirstPersonInteractor, Prompt, Trigger

### PR DESCRIPTION
**Pull request by cpimhoff**
Added these custom editors in.
These editors aren't that complex, but now they have warnings when stuff goes wild!

### Prompt
Basically just removed the 'script' link.
Prompt Editor

### First Person Interactor
Shown with errors currently blasting.
First Person Interactor Editor

### Trigger
Shown with all its error types on blast!
Trigger Interaction Editor

### CODE REVIEW

File: Project/Assets/Editor/CustomInspectors/PromptEditor.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	{
+		prompt = (Prompt)target;
+
+		GUIContent label = new GUIContent ("Prompt Text", "Text displayed when a player can interact with this object.");
+		prompt.promptText = EditorGUILayout.TextField (label, prompt.promptText);
+
+		if (prompt.promptText == null || prompt.promptText == "")
+		{
+			// I feel like there should be a warning here, but I don't know what to write
+			// PrairieGUI.warningLabel("?")
```
Comment (grahamearley) [inline Project/Assets/Editor/CustomInspectors/PromptEditor.cs:19]: Maybe a warning if the text hasn't changed -- i.e. if the prompt text is still the default

Comment (grahamearley) [reply Project/Assets/Editor/CustomInspectors/PromptEditor.cs:19]: Could say "Looks like you didn't update this prompt from the default. A more descriptive prompt makes your game easier to understand" or something

**Review by grahamearley — APPROVED** [review]
Overall, a very pleasant and readable PR. I say let's merge it.
💯 / 💯

**Review by shoemakerc — APPROVED** [review]
Straight peachy.

**Pull request merged**